Sources/UI/Checkout/CheckoutStepsViewController.swift

```diff
@@ -237,7 +237,7 @@ final class CheckoutStepsViewController: UIHostingController<CheckoutView> {
             }
             if let receiptLink = userInfo["receiptLink"] as? SnabbleCore.Link, let url = URL(string: receiptLink.href) {
                 let detailViewController = ReceiptsDetailViewController(orderId: url.lastPathComponent, projectId: viewModel.shop.projectId)
-                present(detailViewController, animated: true)
```

Comment:
Soll es nun auch in einer Stack Navigation angezeigt werden?

Sollten wir nicht eher den `detailViewController` mit einem `UINavigationController` Wrappen, damit es weiterhin Modal angezeigt wird?